projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -155,6 +155,7 @@
       align-items: center;
       height: header-variables.$clr-header-height;
       text-decoration: none;
+      gap: 0;
```

Comment:
Is this (and the other gap values in vertical-nav) necessary? I'm not seeing a change when manipulating them in the preview

Reply:
They are necessary in core themes light, dark and high contrast.

The preview is not changing because uses hamburger icon which is not part of the link. The link icon is actually hidden you have to set it to visible.

The new implementation is introducing `gap` as maintaining spacing (8px) between parts of a button icons and text. The header and in other places are a special case that icons and text should not have gap spacing between them. The spacing is handled in different way. Adding `gap: 0` allow the component to use button in their special way.

Reply:
In header specifically the gap contradicts with specifically this `margin-right`: https://github.com/vmware-clarity/ng-clarity/blob/main/projects/angular/src/layout/nav/_header.clarity.scss#L170